lib/aws/active_job/sqs/cli_options.rb

```diff
@@ -90,8 +90,10 @@ def self.threads_option(opts, out)
           end
         end
 
-        def self.queue_option(opts, out)
-          opts.on('-q', '--queue STRING', '[Required] Queue to poll') { |a| out[:queue] = a }
+        def self.queues_option(opts, out)
+          opts.on('-q', '--queue STRING', 'Queue(s) to poll. You may specify this argument multiple times to poll multiple queues.  If not specified, will start pollers for all queues defined.') do |a|
```

Comment:
--queues with a comma separated string is a possibility

Reply:
I prefer multiple `--queue` options personally and I think is often more the standard way to provide multiple arguments.  Eg: `ruby`/`irb` allows multiple `-r` flags, eg `irb -I./ -r req1.rb -r req2.rb`